todo_app/data/trello_items.py

```diff
@@ -6,23 +6,18 @@
 from todo_app.data.item import Item
 
 dotenv.load_dotenv()
```

Comment:
Can we delete this dotenv step? This could accidentally cause your tests to load real credentials if we're not careful, which puts us at risk of tests triggering real activity if some mocking/patching is incorrectly set up!

Reply:
(Not a problem as it stands, because they get overridden in time by your tests, but I'd still be a little wary)